tutorials/javaFxPart1.md

```diff
@@ -94,6 +94,21 @@ public class Launcher {
 }
 ```
 
+Now that we have changed the entry point to our application, we need to update the `mainClass` 
+attribute in `build.gradle` accordingly to point at the `Launcher` class.
+
+```groovy{highlight-lines="3-5"}
+// ...
+
+application {
+   mainClass.set("Launcher")
+}
+
+// ...
+```
+Now, if we want to run the application, we can use the command `./gradlew run` if we want to use 
```

Comment:
when doing this myself when i was taking 2103T, i didnt know where to run ./gradlew run. and had to google around and ask on forums until i found out we can do `run anything` on intellij on mac by double pressing CTRL. should we add this into the guide?

Reply:
https://github.com/nus-cs2103-AY2324S2/forum/issues/406

Reply:
This is mentioned in the Gradle tutorial. https://github.com/se-edu/guides/blob/7f887785a6e009992f291b50752ac88ebb4a8aef/tutorials/gradle.md?plain=1#L97

Reply:
ah missed that. i guess its up to u to decide if its worth to mention it again.

Reply:
The decision to add this line was to demonstrate the build.gradle was updated and should be working now (and should work with `./gradlew run`, and to highlight the new entry point into the application, which would be `Launcher.java`, and these two methods of running the application are highlighted in the line.

I don't think it would be a good idea to add in the <kbd>Ctrl</kbd> tip as the main focus of this guide is not to teach users how to use the features of their IDE, but instead to show users how their code evolves as JavaFX is incorporated into their application.